src/runtime_functions.js

```diff
@@ -4,6 +4,16 @@
  * SPDX-License-Identifier: MIT
  */
 
+// This gives correct answers for everything less than 2^{14} = 16384
+// I hope nobody is contemplating functions with 16384 arguments...
+function uleb128Encode(n){
+  if(n < 128) {
+      return [n];
+  }
+  return [(n % 128) + 128, n >> 7];
```

Comment:
```suggestion
  return [(n % 128) | 128, n >> 7];
```
(bitwise or seems a little clearer here, as we are basically adding a bit)